pkg/cortex/cortex.go

```diff
@@ -173,6 +174,10 @@ func (c *Config) Validate(log log.Logger) error {
 		return err
 	}
 
+	if c.HTTPPrefix != "" && !strings.HasPrefix(c.HTTPPrefix, "/") {
+		return errors.New("http_prefix should be empty or start with /")
```

Comment:
I wouldn't mention the YAML config name in the error, considering we also support CLI flags. I would just be generic (it's the pattern we've used in most validation errors).

```suggestion
		return errors.New("HTTP prefix should be empty or start with /")
```